clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -967,6 +975,43 @@ private static void applyValueSchemaToStore(CommandLine cmd) throws Exception {
     printObject(valueResponse);
   }
 
+  private static void applyValueSchemaToZK(CommandLine cmd) throws Exception {
+    String store = getRequiredArgument(cmd, Arg.STORE, Command.ADD_SCHEMA_INTO_ZK);
+    String cluster = getRequiredArgument(cmd, Arg.CLUSTER, Command.ADD_SCHEMA_INTO_ZK);
+    String veniceZookeeperUrl = getRequiredArgument(cmd, Arg.VENICE_ZOOKEEPER_URL, Command.ADD_SCHEMA_INTO_ZK);
+    String valueSchemaFile = getRequiredArgument(cmd, Arg.VALUE_SCHEMA, Command.ADD_SCHEMA_INTO_ZK);
+    int valueSchemaId = Utils.parseIntFromString(
+        getRequiredArgument(cmd, Arg.VALUE_SCHEMA_ID, Command.ADD_SCHEMA_INTO_ZK),
+        Arg.VALUE_SCHEMA_ID.toString());
+    String valueSchemaStr = readFile(valueSchemaFile);
+    verifyValidSchema(valueSchemaStr);
+    Schema newValueSchema = Schema.parse(valueSchemaStr);
+
+    HelixSchemaAccessor schemaAccessor =
```

Comment:
This is actually tricky.  ZK will require SSL at some point, and today SSL is set up for helix objects via a JVM system argument.

Reply:
What we can probably do is require (or optionally require?) ssl arguments for keystore/truststore and then if they're present set up the jvm system variable before creating the necessary Helix objects.  That should make sure the SSL session is set up.

Reply:
Oh, I see. Is SSL already enforced in ZK now? If so, I can make it a required argument.

Reply:
@ZacAttack Could you please share the jvm variable names that I should check for? I couldn't find them in the code base.